ApiBundle/Transformer/ApiClientCollectionTransformer.php

```diff
@@ -13,15 +13,15 @@
 class ApiClientCollectionTransformer extends AbstractTransformer
 {
     /**
-     * @param Collection $mixed
+     * @param Collection $apiClientCollection
      *
      * @return FacadeInterface
      */
-    public function transform($mixed)
+    public function transform($apiClientCollection)
```

Comment:
Can we use this refacto to enhance the stability ? As the param ($mixed) has no explicit type in the interface, we should at least check here that the param has the expected type in that context and raise an error if not.
This should be done for all transformers.